### PR TITLE
Added AWS Marketplace option in pricing page.

### PR DESCRIPTION
**Pull request by kmikawa**
Added AWS Marketplace option in pricing page.
Added a custom TextBlock for new design.
This is for stage deployment only.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/demo.md [hunk 1]
```diff
@@ -10,5 +10,5 @@ keywords:
   - C++
   - Scripting
 title: Photoshop API Demo
-frameSrc: https://easybake.adobedice.com?ref=home
+frameSrc: https://easybake-stage.adobedice.com?ref=home
 ---
```
Comment (khound) [inline src/pages/demo.md:14]: How we take care of environments? I mean when we deploy to prod?

File: src/pages/signup.md [hunk 2]
```diff
@@ -10,5 +10,5 @@ keywords:
   - C++
   - Scripting
 title: Photoshop API Get Started
-frameSrc: https://easybake.adobedice.com?ref=signup
+frameSrc: https://easybake-stage.adobedice.com?ref=signup
 ---
```
Comment (khound) [inline src/pages/signup.md:14]: Same question regarding environment?

**Review by khound — APPROVED** [review]

File: src/pages/demo.md (hunk 1 above)
Comment (kmikawa) [reply src/pages/demo.md:14]: There is no environment variable set up for this repo, so we need to deploy separately.  This branch is for stage only.

**Comment by khound** [comment]
👍

**Pull request merged**